### ISSUE TITLE
autocomplete for normal fields

### ISSUE THREAD

**Issue by maariioo** [geex-arts/django-jet#256]
Hello.
I have working autocomplete for both ForeignKey and ManyToManyField fields.
But I don't know how to have this feature working for the rest of fields (e.g CharField,DateField..)

**Comment by SalahAdDin** [comment]
Did you try appliying the same method?

**Comment by maariioo** [comment]
yes, I try, but I see that comboboxes are populated with all data, and ajax are not sent when I write

**Comment by SalahAdDin** [comment]
Can youput here the traceback?

**Comment by maariioo** [comment]
I don't know how to get a traceback for this issue..

**Comment by SalahAdDin** [comment]
Review in browser: use developer tools.

**Comment by maariioo** [comment]
I can't send you a traceback because there is not a javascript exception. The dropdown is initialy filled with all values (see jet demo url for adress)

**Comment by SalahAdDin** [comment]
We need make a traceback.
